### DISCUSSION TITLE
Safe Exam Browser quitting unexpectedly in Mac OS

### DISCUSSION THREAD

**Discussion by daevielle** [SafeExamBrowser/seb-mac#188] [Q&A]
Hello! I've been trying to take exams through the Safe Exam Browser but while I am using the browser it quits unexpectedly. This happens multiple times. I attached a picture of what appears when the browser quits. What should I do?

ps. I am using mac os, 11 big sur. 
<img width="772" alt="Screen Shot 2022-03-03 at 4 22 47 PM" src="https://user-images.githubusercontent.com/100840045/156544580-c07befca-c304-48f1-9e3d-ec3e25afd77e.png">

**Comment by danschlet** [comment]
That's most likely this issue: https://github.com/SafeExamBrowser/seb-mac/issues/176
The issue should be fixed in SEB 3.1, you can try the preview version.
